src/datahike/api.cljc

```diff
@@ -262,6 +263,10 @@
              Query passed as map needs vectors as values. Query can not be passed as list. The 1-arity function takes a map with the arguments :query and :args and optionally the additional keys :offset and :limit."}
   q dq/q)
 
+(def ^{:arglists '([query & args] [arg-map])
```

Comment:
We might need some info on the execution details since it is not so clear what the different keywords mean. Maybe an example and then some explanations in the docs would be helpful.

Reply:
It's exactly the same as for q, so I don't think we need extra documentation. I added a note about that behaviour in the docstring though.